Python/Product/Analysis/Parsing/Ast/DecoratorStatement.cs

```diff
@@ -27,9 +27,7 @@ public class DecoratorStatement : Statement {
             _decorators = decorators;
         }
 
-        public IList<Expression> Decorators {
-            get { return _decorators; }
-        }
+        public Expression[] Decorators => _decorators;
```

Comment:
This breaks a static analysis rule about returning references to arrays through public API